schools/aggiecorps.yml

```diff
@@ -5,6 +5,7 @@ theme:
   hover: '#880000'
   background: 'https://static.gradebook.app/sYbR9JXKTI/aggiecorps.jpg'
 gpa: true
+partner: true
```

Comment:
Has this been implemented in client / the collaborate PR?

Reply:
Yes, but we can change it if needed

Reply:
Okay cool just checking